ui/common/src/socket.ts

```diff
@@ -95,10 +87,12 @@ export default class StrongSocket {
     version: number | false,
     settings: Partial<Settings> = {},
   ) {
+    console.log('wtf');
     this.settings = {
       receive: settings.receive,
       events: settings.events || {},
       params: {
+        sri: site?.sri,
```

Comment:
`site` shouldn't need the `?` as it should always exists according to
```ts
declare const site: Site;
```

There's no `?` used in this file line 55:
```ts
export default class StrongSocket {
  pubsub = site.pubsub;
```

Reply:
I didn't think so either, but did we not run into a problem at one point in some environment where socket was dying because there was no window.site? or was it just site.siteI18n that was null?

Reply:
> I didn't think so either, but did we not run into a problem at one point in some environment where socket was dying because there was no window.site? or was it just site.siteI18n that was null?

https://github.com/lichess-org/lila/issues/14666 ?